### PR TITLE
Added proplot

### PR DESCRIPTION
**Pull request by hmaarrfk**
I removed the vendored fonts. It seems that matplotlib will simply emit an issue at this stage warnign the user about the missing font and falling back to a predefined default.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@lukelbd do you use conda? do you want to maintain this as well?

**Comment by hmaarrfk** [comment]
@lukelbd do you have a license to distribute all the fonts you typically vendor in the wheel? If so, are you in the position to extend it to this package?

**Comment by lukelbd** [comment]
Thanks for starting this thread -- I do use conda, would love to allow people to `conda install` ProPlot.

This was originally just a personal project so I didn't think very carefully about licensing. I just wanted to have the fonts I use on my macbook available on the Linux server I also work on. In a new version I'll remove all of the fonts that weren't obtained from Google Fonts; please wait until then (will let you know).

Is there anything else I need to do?

**Comment by hmaarrfk** [comment]
Do you think it is OK to just ship this as is with the fonts removed? it issues warnings, but can help users get started?

Can you please express your intent to be a maintainer? A straight "Yes I would like to maintain this package" or "No I do not want to maintain this package" really helps since you will get spam from conda forge ;)

We can wait for a new release if your maintenance help is conditioned on that.

**Comment by lukelbd** [comment]
Just released a new version, feel free to ship (you can now remove the `rm -rf proplot/fonts` line, all the fonts in that folder are from Google Fonts).

I was slightly very worried about publishing a package with prioprietary fonts, wanted to get this done right away. I may also remove the older  verion 0.0-0.2 PyPi releases at some point.

And yep, I'd like to maintain this package!

**Comment by hmaarrfk** [comment]
Awesome, also, i'm glad you updated your comment about the absence of Helvetica by matplotlib ;)

**Comment by lukelbd** [comment]
Haha my bad, I must have been feeling particularly snarky when I wrote that. Still adjusting to proplot going from my quirky little side project to something people actually use

**Comment by hmaarrfk** [comment]
@conda-forge/staged-recipes kind ping

File: recipes/proplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - proplot
+
+about:
+  home: https://proplot.readthedocs.io/en/latest/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'A comprehensive, easy-to-use matplotlib wrapper for making beautiful, publication-quality graphics.'
+  dev_url: https://github.com/simplejson/simplejson
```
Comment (npavlovikj) [inline recipes/proplot/meta.yaml]: I think the `dev_url` should be https://github.com/lukelbd/proplot?

Comment (hmaarrfk) [reply recipes/proplot/meta.yaml]: Thanks!

File: recipes/proplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 75e6c871bb628ed2879eb1a997c159a405ba69655e8010489f50f74877813b7a
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - rm -rf proplot/fonts
```
Comment (npavlovikj) [inline recipes/proplot/meta.yaml]: I just saw the developer has commented earlier that this `rm` line is not needed anymore. Do you want to try removing it? I am ok merging the PR regardless.

Comment (hmaarrfk) [reply recipes/proplot/meta.yaml]: It still vendors quite a few fonts.

We would have to make sure that all those licenses are valid.

I think for now, it is best to simply avoid the fonts folder due to licensing confusion

image

**Comment by npavlovikj** [comment]
Thanks for the clarification and contribution, @hmaarrfk 🎉

**Comment by hmaarrfk** [comment]
Thanks!

**Comment by lukelbd** [comment]
Hey guys, thanks!

The current fonts are all from Google Fonts, which means they are [free for individual and commercial use](https://developers.google.com/fonts/faq). Any way we can remove the `rm` line (should I open a new issue)?

**Comment by lukelbd** [comment]
I see that matplotlib [includes the individual licenses](https://github.com/matplotlib/matplotlib/tree/master/lib/matplotlib/mpl-data/fonts/ttf) for each font. I'll add the license files to `proplot/fonts` then ping this thread again.

**Comment by npavlovikj** [comment]
@lukelbd , "proplot" is now available on the conda-forge channel, and there is a feedstock generated for it, https://github.com/conda-forge/proplot-feedstock. 

All further changes in the recipe for "proplot" need to be done using the feedstock now. Since you are listed as a maintainer, you can modify the recipe yourself, following the instructions provided here, https://conda-forge.org/#update_recipe.

If you have any issues with updating the feedstock, please feel free to ping me or Mark - I would be more than happy to help.

**Comment by lukelbd** [comment]
Got it; thanks again.

**Pull request merged**